cmd/cometbft/commands/run_node.go

```diff
@@ -94,8 +89,8 @@ func NewRunNodeCmd(nodeProvider nm.Provider) *cobra.Command {
 		Aliases: []string{"node", "run"},
 		Short:   "Run the CometBFT node",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			if err := checkGenesisHash(config); err != nil {
-				return err
+			if len(genesisHash) != 0 {
+				config.Storage.GenesisHash = genesisHash
```

Comment:
Would it be helpful to do some extra validation ? like if `len(genesisHash) == 64`.

Reply:
Maybe add the check that was done below - || config.Genesis == ""